db/migrate/20180510090438_create_addresses.rb

```diff
@@ -0,0 +1,22 @@
+class CreateAddresses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :addresses, id: false do |t|
+      t.uuid :id, primary_key: true
+      t.string :street_address
+      t.string :colony
+      t.string :village
+      t.string :district
+      t.string :state
+      t.string :country
+      t.string :pin
```

Comment:
This all is very India-specific. We should model these fields to be more generic (`postal_code` for example). Let's discuss.